config/locales/en.yml

```diff
@@ -1769,6 +1769,7 @@ en:
     contrast: Mastodon (High contrast)
     default: Mastodon (Dark)
```

Comment:
Probably not in this PR - but a possible future improvement here would be a rename in this area so that the collection becomes: `[high-contrast, mastodon-light, mastodon-dark, system]` (or something along those lines).

I could definitely see having a theme named "default" - which is not the default! - becoming confusing.

We'd need a migration to update settings, and some code renames as well.

Reply:
We could do this, but it will need to be in another PR with a migration in the various places where this is used (instance settings and user settings). I suspect it might break other things that rely on the `default` string somewhere (like the emoji code at the moment is relying on `mastodon-light`)

Reply:
Yes, definitely out of scope for this PR, and noted that we'd have to be diligent on rename.